README.md

```diff
@@ -469,35 +465,43 @@ Fully customisable.  |
 :star2: | Name | Description | 🌍
 --- | --- | --- | ---
 2 | [@mkatircioglu](https://github.com/mkatircioglu)/[**dotfiles**](https://github.com/mkatircioglu/dotfiles) |  |
-### Crystal #
```

Comment:
👍 Thanks for clarification, I didn't know the way you do that. Do you use this package to generate? https://www.npmjs.com/package/package-json-to-readme